src/test/java/org/folio/rest/impl/UsersAPIIT.java

```diff
@@ -121,8 +121,28 @@ static void postTenant() {
     given().when().get("/_/tenant/" + id).then().statusCode(200).body("$", not(hasKey("error")));
   }
 
-  void facets(int limit) {
+  static void userExists(String id) {
+    given().
+    when().get("/users/" + id).
+    then().statusCode(200);
+  }
+
+  static void userDoesntExist(String id) {
+    given().
+    when().get("/users/" + id).
+    then().statusCode(404);
+  }
+
+  static void postUser(String id, String username) {
```

Comment:
I believe the intent is to create a user. I think it is a detail that this helper is deliberately trying to hide that it is done via a POST request.

```suggestion
  static void createUser(String id, String username) {
```

Reply:
This is an integration test that also tests the POST /user API. This helper doesn't try to hide anything but to avoid duplicate code.

Reply:
> This is an integration test that also tests the POST /user API. This helper doesn't try to hide anything but to avoid duplicate code.

I don't understand why the type of tests changes the value of expressing intent to improve the readability of the code.

Are you telling me that you are rejecting my suggestion as irrelevant?

Reply:
The method name `postUser` indicates that the POST /user API is used to create the user.
The method name `createUser` doesn't and the `createUser` method may use any implementation to create the user, for example an SQL query.
`postUser` might be used for tests we may add in the future where the POST /user API is the API under test.
Therefore I want the method name to be `postUser` and not `createUser`.

Reply:
> The method name postUser indicates that the POST /user API is used to create the user.

Why, within this test, is the specifics of the mechanism for user creation important? 

To me, the intention is to create a user, not to exercise a particular endpoint. The goal of this part of the state is to arrange the required state for the rest of the test. How that is done is less relevant than what is trying to be achieved.

> postUser might be used for tests we may add in the future where the POST /user API is the API under test.

Hopefully there are already tests for that endpoint already. I don't think speculation about future use is a good argument for why it is more important to be specific about the mechanism rather than the intention.

> Therefore I want the method name to be postUser and not createUser.

I understand that you disagree with my feedback. It is your change and so it is your preference. You can reject the feedback.